scalajslib/src/ScalaJSModule.scala

```diff
@@ -99,6 +99,50 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
     )
   }
 
+  private def isLinkJSSupported = T {
+    scalaJSVersion() match {
+      case v if v.startsWith("0.") => false
+      case v if v < "1.3" => false
+      case _ => true
+    }
+  }
```

Comment:
```suggestion
  private def isLinkJSSupported = T {
    scalaJSVersion() >= "1.3"
  }
```

Also try if `T.task` work, since you don't need to save this to a file.

Reply:
After checking that you can use `org.scalajs.ir.ScalaJSVersions.current` inside the worker, now I think you probably don't need to make this check here, so you can call the worker directly.